core/domain/story_domain.py

```diff
@@ -576,12 +627,14 @@ def create_default_story(cls, story_id):
         Returns:
             Story. The Story domain object with the default values.
         """
+        # Initial node id for a new story.
+        DEFAULT_INITIAL_NODE_ID = NODE_ID_PREFIX + '1'
         story_contents = StoryContents(
             [
                 StoryNode.create_default_story_node(
-                    feconf.DEFAULT_INITIAL_NODE_ID)
+                    DEFAULT_INITIAL_NODE_ID)
             ],
-            feconf.DEFAULT_INITIAL_NODE_ID, 2)
+            DEFAULT_INITIAL_NODE_ID, NODE_ID_PREFIX + '2')
```

Comment:
StoryNode.get_incremented_node_id(initial_node_id)


(general principle: always try to pin things down to a single source of truth if they're related, rather than have two seemingly-independent pieces of data)

Reply:
Done.